### PR TITLE
Super Skelemania

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Super Skelemania

## Game Information
- **Title**: Super Skelemania
- **URL**: https://benal.itch.io/super-skelemania

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [ ] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — DISMISSED** [review]

File: ports/superskelemania/Super Skelemania.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+# device_info.txt will be included by default
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+export PORT_32BIT="Y"
```
Comment (Cebion) [inline ports/superskelemania/Super Skelemania.sh:21]: Has to be before mod_cfw line

File: ports/superskelemania/Super Skelemania.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+$GPTOKEYB "gmloader" -c ./superskelemania.gptk &
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+pm_platform_helper $GAMEDIR/gmloader
```
Comment (Cebion) [inline ports/superskelemania/Super Skelemania.sh:57]:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

File: ports/superskelemania/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "genres": [
+      "platformer"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
+      "aarch64"
```
Comment (Cebion) [inline ports/superskelemania/port.json:27]: gmloader is 32 bit
```suggestion
      "armhf"
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/superskelemania/superskelemania/game.apk
Comment (cdeletre) [inline ports/superskelemania/superskelemania/game.apk:1]: Remove this files from the apk

```
unzip -l game.apk
Archive:  game.apk
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  02-14-2024 20:32   lib/
        0  02-14-2024 20:32   lib/armeabi/
   312924  02-14-2024 20:32   lib/armeabi/libopenal.so
  4878276  02-14-2024 20:32   lib/armeabi/libyoyo.so
        0  02-14-2024 20:32   lib/armeabi-v7a/
   288356  02-14-2024 20:32   lib/armeabi-v7a/libopenal.so
  4689844  02-14-2024 20:32   lib/armeabi-v7a/libyoyo.so
        0  02-14-2024 20:32   lib/mips/
   414556  02-14-2024 20:32   lib/mips/libopenal.so         <=====
  6718636  02-14-2024 20:32   lib/mips/libyoyo.so           <=====
        0  02-14-2024 20:32   lib/x86/
   316964  02-14-2024 20:32   lib/x86/libopenal.so          <=====
  5943740  02-14-2024 20:32   lib/x86/libyoyo.so            <=====
---------                     -------
 23563296                     13 files
```

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/superskelemania/superskelemania/game.apk
Comment (tabreturn) [reply ports/superskelemania/superskelemania/game.apk:1]: Done. Thx

File: ports/superskelemania/Super Skelemania.sh (hunk 1 above)
Comment (tabreturn) [reply ports/superskelemania/Super Skelemania.sh:21]: Done. Thx

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

**Pull request merged**